src/query/api/v1/handler/graphite/render_parser.go

```diff
@@ -83,21 +84,28 @@ type RenderRequest struct {
 }
 
 // ParseRenderRequest parses the arguments to a render call from an incoming request.
-func ParseRenderRequest(r *http.Request) (RenderRequest, error) {
-	var (
-		p   RenderRequest
-		err error
-		now = time.Now()
-	)
+func ParseRenderRequest(
+	r *http.Request,
+	opts options.HandlerOptions,
+) (RenderRequest, *storage.FetchOptions, error) {
+	fetchOpts, err := opts.FetchOptionsBuilder().NewFetchOptions(r)
+	if err != nil {
+		return RenderRequest{}, nil, err
+	}
 
-	if err = r.ParseForm(); err != nil {
-		return p, err
+	if err := r.ParseForm(); err != nil {
```

Comment:
```suggestion
	if err = r.ParseForm(); err != nil {
```
Think we already have err defined, since this fails fast anyway, shouldn't be any worry with shadowing